source/documentation/index.md

```diff
@@ -1,21 +1,5 @@
-# Hello, World!
+# Data.gov.uk technical documentation
```

Comment:
We've always stylised the service name as "data.gov.uk" in the past (lowercase d), even when it is at the start of a sentence.  Would this be the correct thing to continue doing?